test/services/licences/view-licence.service.test.js

```diff
@@ -173,6 +174,130 @@ describe('View Licence service', () => {
         expect(result.warning).to.be.null()
       })
     })
+
+    describe('and it has a source of supply', () => {
+      beforeEach(() => {
+        fetchLicenceResult = _testLicence()
+        fetchLicenceResult.permitLicence = {
+          purposes: [{
+            purposePoints: [{
+              point_source: {
+                NAME: 'SURFACE WATER SOURCE OF SUPPLY'
+              }
+            }]
+          }]
+        }
+        Sinon.stub(FetchLicenceService, 'go').resolves(fetchLicenceResult)
+      })
+
+      it('will return the source of supply for use in the licence summary page', async () => {
+        const result = await ViewLicenceService.go(testId)
+
+        expect(result.sourceOfSupply).to.equal('SURFACE WATER SOURCE OF SUPPLY')
+      })
+    })
+
+    describe('and it does not have a source of supply name', () => {
+      beforeEach(() => {
+        fetchLicenceResult = _testLicence()
+        fetchLicenceResult.permitLicence = {
+          purposes: [{
+            purposePoints: [{
+              point_source: {}
+            }]
+          }]
+        }
+        Sinon.stub(FetchLicenceService, 'go').resolves(fetchLicenceResult)
+      })
+
+      it('will return the source of supply for use in the licence summary page', async () => {
```

Comment:
Just checking this, and the others like it underneath are correct?

It is returning a property but it is `null`. And we are not distinguishing the message from the test where a value is returned. Should it be something like

```suggestion
      it('will return null for the source of supply', async () => {
```